src/main/java/zabi/minecraft/extraalchemy/mixin/common/BrewingStandMixin.java

```diff
@@ -26,6 +26,8 @@ public abstract class BrewingStandMixin extends LockableContainerBlockEntity imp
 	
 	private static final TagKey<Block> HEAT_SOURCE_TAG = TagKey.of(Registry.BLOCK_KEY, LibMod.id("heat_source"));
 	private static final TagKey<Block> HEAT_CONDUCTOR_TAG = TagKey.of(Registry.BLOCK_KEY, LibMod.id("heat_conductor"));
+	private static final int BREWING_STAND_MAX_FUEL_CAPACITY = Math.min(20, ModConfig.INSTANCE.brewingStandFireMaxCapacity);
+	private static final int BREWING_STAND_HEAT_INCREMENT_DELAY_TICKS = 20 * ModConfig.INSTANCE.brewingStandHeatIncrementDelay;
```

Comment:
Having these as constants makes it so that changes to the config are not updated in real time, but require a client restart. They should be moved to local variables declared between lines 41 and 42 of this same file